libs/isograph-react/src/loadable-hooks/useConnectionSpecPagination.ts

```diff
@@ -20,10 +20,27 @@ import { useIsographEnvironment } from '../react/IsographEnvironmentProvider';
 import { useSubscribeToMultiple } from '../react/useReadAndSubscribe';
 import { maybeUnwrapNetworkRequest } from '../react/useResult';
 
-type FirstOrAfter = 'first' | 'after';
-type OmitFirstAfter<TArgs> = keyof Omit<TArgs, FirstOrAfter> extends never
-  ? void | Record<string, never>
```

Comment:
We don't need `Record<string, never>` in `args`, because there's always at least one key `after` or `skip`

Reply:
sgtm